projects/sky-autonumeric/src/modules/autonumeric/autonumeric.directive.ts

```diff
@@ -129,13 +129,11 @@ export class SkyAutonumericDirective
         this.isFirstChange && this.control && this.value !== null;
       if (initializedWithValue) {
         this.isFirstChange = false;
-        this.control.markAsPristine();
+        this.control?.markAsPristine();
```

Comment:
Should this be `this.control!.markAsPristine()` since previously we didn't check for undefined? Using a conditional will create a new uncovered code branch.

Reply:
You're right, `initializedWithValue` checks that it's set.